src/js/components/Meter/Circle.js

```diff
@@ -129,7 +134,7 @@ const Circle = forwardRef((props, ref) => {
         strokeWidth={height}
         fill="none"
         {...strokeProps(background, theme)}
-        strokeLinecap="round"
+        strokeLinecap={round ? 'round' : 'square'}
```

Comment:
Why are we changing this?

Reply:
<img width="136" alt="Screen Shot 2021-08-03 at 1 29 17 PM" src="https://user-images.githubusercontent.com/42451602/128074631-cd483913-913f-4e20-87e2-2004e2749bc5.png">
This is an option in props and I was not taking it into account 
https://v2.grommet.io/meter#round